packages/@okta/vuepress-site/docs/reference/api/zones/index.md

```diff
@@ -293,6 +295,88 @@ curl -X POST \
 }
 ```
 
+### Create a Block List Network Zone
+
+
+<ApiOperation method="post" url="/api/v1/zones" />
+
+Creates a new Block List Network Zone
+
+#### Valid Request Example
+
+```bash
+curl -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "IP",
+  "id": null,
+  "name": "newBlockListNetworkZone",
+  "status": "ACTIVE",
+  "usage": "BLOCKLIST",
+  "created": null,
+  "lastUpdated": null,
+  "gateways": [
+    {
+      "type": "CIDR",
+      "value": "1.2.3.4/24"
+    },
+    {
+      "type": "CIDR",
+      "value": "2.3.4.5/24"
+    }
+  ],
+  "proxies": null
+}' "https://${yourOktaDomain}/api/v1/zones"
+```
+
+#### Successful Response Example
```

Comment:
Are you planning to provide an unsuccessful response example? If not, then this can just be: Response example